P/PulseAudio/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PulseAudio"
+version = v"14.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://freedesktop.org/software/pulseaudio/releases/pulseaudio-14.2.tar.gz", "902dd1928801bb5dc7b121754aa4110ce55768b5dff94a700e7bd58d3f597970")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add gettext
+apk add glib
+apk add orc-compiler
+apk add perl-xml-parser
+apk add bash-completion
+# For some reason, librt fails to get linked correctly, so add a flag
+sed -i -e "s~c_args = .*~c_args = ['-I${includedir}', '-L${libdir}']~" ${MESON_TARGET_TOOLCHAIN}
+sed -i -e "s~c_link_args = .*~c_link_args = ['-lrt', '-Wl,-rpath=${includedir}']~" ${MESON_TARGET_TOOLCHAIN}
```

Comment:
`-Wl,-rpath=${includedir}` looks suspiciously wrong: `-rpath` is for the linker, in `includedir` there are header files

Reply:
Maybe it was a failed experiment...I'll delete it and see what happens